sharkfuser/include/fusili/external_tools.h

```diff
@@ -0,0 +1,6 @@
+#ifndef FUSILI_TOOLS_H
+#define FUSILI_TOOLS_H
```

Comment:
This would be `FUSILI_EXTERNAL_TOOLS_H` going by the LLVM convention for header guards https://llvm.org/docs/CodingStandards.html#header-guard